git/test/fixtures/blame_incremental_2.11.1_plus

```diff
@@ -0,0 +1,33 @@
+82b8902e033430000481eb355733cd7065342037 2 2 1
```

Comment:
Thanks a lot! I am ready to merge!
One last question though: should there or should there not be empty lines in the fixture?